baggers.js

```diff
@@ -14768,8 +14796,7 @@ var data = {
       "name": "Jean Bisutti",
       "bio": "Développeur Java",
       "picture": "https://pbs.twimg.com/profile_images/606213307507351552/IJcDzWZo.jpg",
-      "websites": [
```

Comment:
This is not needed. Could you revert that change?

Reply:
right away !